cogctl/cli/version.py

```diff
@@ -19,22 +19,39 @@ def get_build_version():
     Return the contents of the file `cogctl/GITSHA`. This is generated
     at build-time and contains the truncated Git SHA of the code being
     built.
+    """
+
+    return _load_bundled_file('GITSHA')
+
+
+def get_build_tag():
+    """
+    Return the contents of the file `cogctl/GITTAG`. This is generated
+    at build-time and contains the Git tag or branch name of the code
+    being built. If no file is found then the string `'dev'` is used.
+    """
+
+    return _load_bundled_file('GITTAG', default='dev')
+
+
+def _load_bundled_file(name, default='unknown'):
```

Comment:
Minor nitpick, but this would probably be better named something like `_read_bundled_file` or `_bundled_file_contents`